diff --git a/node_modules/@expo/cli/build/src/start/server/metro/dev-server/createMetroMiddleware.js b/node_modules/@expo/cli/build/src/start/server/metro/dev-server/createMetroMiddleware.js
index 0287ef7..1344865 100644
--- a/node_modules/@expo/cli/build/src/start/server/metro/dev-server/createMetroMiddleware.js
+++ b/node_modules/@expo/cli/build/src/start/server/metro/dev-server/createMetroMiddleware.js
@@ -72,7 +72,7 @@ const metroOpenStackFrameMiddleware = (req, res, next)=>{
 };
 function createMetroStatusMiddleware(metroConfig) {
     return (_req, res)=>{
-        res.setHeader('X-React-Native-Project-Root', metroConfig.projectRoot);
+        res.setHeader('X-React-Native-Project-Root', encodeURIComponent(metroConfig.projectRoot));
         res.end('packager-status:running');
     };
 }
